test/unit/uk/gov/hmrc/individualsbenefitsandcreditsapi/audit/AuditHelperSpec.scala

```diff
@@ -50,84 +53,71 @@ class AuditHelperSpec extends UnitSpec with MockitoSugar {
   val httpExtendedAuditEvent = new DefaultHttpExtendedAuditEvent(
     "individuals-benefits-and-credits-api")
 
-  val apiResponseEvent = new ApiResponseEvent(httpExtendedAuditEvent)
-  val apiFailureEvent = new ApiFailureEvent(httpExtendedAuditEvent)
-  val ifApiResponseEvent = new IfApiResponseEvent(httpExtendedAuditEvent)
-  val ifApiFailureEvent = new IfApiFailureEvent(httpExtendedAuditEvent)
-  val scopesAuditEvent = new ScopesAuditEvent(httpExtendedAuditEvent)
+  val workingTaxCreditResponse = Seq(WtcApplication(None, Seq(WtcAward(None, None, None, None, None))))
+  val childTaxCreditResponse = Seq(CtcApplication(None, Seq(CtcAward(None, None, None, None))))
+  val ifResponse = IfApplications(Seq(IfApplication(None, None, None, None, None)))
 
-  val auditHelper = new AuditHelper(
-    auditConnector, apiResponseEvent, apiFailureEvent, ifApiResponseEvent, ifApiFailureEvent, scopesAuditEvent
-  )
+  val auditHelper = new AuditHelper(auditConnector)
 
   "Auth helper" should {
 
     "auditAuthScopes" in {
 
       Mockito.reset(auditConnector)
 
-      val captor = ArgumentCaptor.forClass(classOf[ExtendedDataEvent])
-
-      auditHelper.auditAuthScopes(matchId, scopes.get, request)
-
-      verify(auditConnector, times(1)).sendExtendedEvent(captor.capture())(any(), any())
-
-      val result = Json.parse(
-        """
-          |{
-          |  "apiVersion": "2.0",
-          |  "matchId": "80a6bb14-d888-436e-a541-4000674c60aa",
-          |  "scopes": "test",
-          |  "method": "GET",
-          |  "deviceID": "-",
-          |  "ipAddress": "-",
-          |  "referrer": "-",
-          |  "Authorization": "-",
-          |  "input": "Request to /",
-          |  "userAgentString": "-"
-          |}
-          |""".stripMargin)
-
-      val capturedEvent = captor.getValue
-      capturedEvent.asInstanceOf[ExtendedDataEvent].auditSource shouldEqual "individuals-benefits-and-credits-api"
-      capturedEvent.asInstanceOf[ExtendedDataEvent].auditType shouldEqual "AuthScopesAuditEvent"
-      capturedEvent.asInstanceOf[ExtendedDataEvent].detail shouldBe result
+      val captor = ArgumentCaptor.forClass(classOf[ScopesAuditEventModel])
+
+      auditHelper.auditAuthScopes(matchId, scopes, request)
+
+      verify(auditConnector, times(1)).sendExplicitAudit(eqTo("AuthScopesAuditEvent"),
+        captor.capture())(any(), any(), any())
+
+      val capturedEvent = captor.getValue.asInstanceOf[ScopesAuditEventModel]
+      capturedEvent.asInstanceOf[ScopesAuditEventModel].apiVersion shouldEqual "1.0"
```

Comment:
capturedEvent is already instance of ScopesAuditEventModel (from line above)

Reply:
Good spot